README.md

```diff
@@ -1,4 +1,4 @@
-# Salesforce DX Project: Next Steps
+# Salesforce DX Project: Next Stepss
```

Comment:
```suggestion
# Salesforce DX Project: Next Steps
```

why would you add another s to this word?

Reply:
It was in the AC